File: C/beginner/1149.c
// https://judge.beecrowd.com/pt/problems/view/1149
// Somando Inteiros Consecutivos

// Libraries
#include <stdio.h>

#include <stdio.h>

// Starting Code
int main() {
    
    int usr_input1, usr_input2, sum = 0;

    // Getting first number
    scanf("%d", &usr_input1);

    // Testing second number
    do {
        scanf("%d", &usr_input2);
    } while (usr_input2 <= 0);

    // Doing the math
    for (int i = 0; i < usr_input2; i++) {
        sum += usr_input1 + i;
    }

    // Showing results
    printf("%d\n", sum);
    
    return 0;
}

// By Fernando Serra
// https://github.com/fernandosserra

File: Python/beginner/1149.py
# https://judge.beecrowd.com/pt/problems/view/1149
# Somando Inteiros Consecutivos

# Requesting
values = list(map(int, input().split()))
usr_input1 = values[0]

# Testing the second value
usr_input2 = next((x for x in values[1:] if x > 0), None)

while usr_input2 is None or usr_input2 <= 0:
    usr_input2 = int(input())

result = 0

# Doing the math
for i in range(usr_input2):
    result += usr_input1 + i

print(result)

# By Fernando Serra
# https://github.com/fernandosserra